src/Atmos/Model/turbulence.jl

```diff
@@ -0,0 +1,138 @@
+#### Turbulence closures
+abstract type TurbulenceClosure
```

Comment:
Would be good to distinguish this in some way from the 'larger-scale' turbulence and convection schemes we have.  Maybe something like `SGS_model` or `SGS_turbulence` is better, though ambiguous in itself, because the convection scheme is also an SGS model.

Reply:
I'm familiar with Turbulence Closure from engineering. I think that it will be easy to adjust this name later on if we'd like